front/migrations/20240411_agent_config_instructions.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  AgentConfiguration,
+  AgentGenerationConfiguration,
+} from "@app/lib/models";
+import logger from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+const backfillAgentConfiguration = async (
+  agent: AgentConfiguration,
+  execute: boolean
+): Promise<void> => {
+  const prompt = agent.generationConfiguration?.prompt;
+  if (!prompt) {
+    logger.info(`Skipping agent (no generation configuration) ${agent.id}`);
+    return;
+  }
+
+  if (execute) {
+    await agent.update({
+      instructions: prompt,
+    });
+  }
+};
+
+const backfillAgentConfigurations = async (execute: boolean) => {
+  // Fetch all agents that have no instructions
+  const agents = await AgentConfiguration.findAll({
+    where: {
+      instructions: null,
+    },
+    include: [
```

Comment:
I'm thinking this might be very fat in production -> I could decided to load all AgentGenerationConfiguration and then using their agentConfigurationId I update the instructions field.